api/swagger.yml

```diff
@@ -1299,6 +1299,9 @@ components:
         force:
           type: boolean
           default: false
+        if_absent:
+          type: boolean
+          default: false
```

Comment:
We use the [HTTP header If-Match](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Match) elsewhere in this API, and I believe that we should continue to do the same and use the [HTTP header If-None-Match](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match) here.  The only legal value will be `*`, of course.

Reply:
Done + removed deprecation for post object